### PR TITLE
(chore) Use pbr for packaging for automated versioning with git tag, move to setup.cfg

### PR DESCRIPTION
**Pull request by guenp**
Using some tools to (hopefully) make our lives easier :)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    distutils
+
+[options]
+install_requires=
+    dbt-common>=1,<2
+    dbt-adapters>=1,<2
+    duckdb>=1.0.0
+    # add dbt-core to ensure backwards compatibility of installation, this is not a functional dependency
+    dbt-core>=1.8.0
+    pbr>=6.0.0
```
Comment (jwills) [inline setup.cfg:31]: Wait is this right? The user of the package doesn't need to install `pbr`, right? It's just a dev dependency?

Comment (guenp) [reply setup.cfg:31]: Ah good point
I was using `pbr` to get the package version in `version.py` but I can just get that with `importlib` instead. will update

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = dbt-duckdb
+author = Josh Wills
+author-email = joshwills+dbt@gmail.com
```
Comment (SaravananSathyanandhaQC) [inline setup.cfg:4]: @guenp @jwills should this be `author_email`? I think `author-email` is deprecated

Comment (jwills) [reply setup.cfg:4]: https://github.com/duckdb/dbt-duckdb/pull/428

**Pull request merged**